core/new-gui/src/app/workspace/component/result-panel/visualization-frame/visualization-frame.component.ts

```diff
@@ -12,27 +15,47 @@ import { VisualizationFrameContentComponent } from "../../visualization-panel-co
   selector: "texera-visualization-frame",
   templateUrl: "./visualization-frame.component.html",
   styleUrls: ["./visualization-frame.component.scss"],
+  providers: [{ provide: NZ_ICONS, useValue: [FullscreenExitOutline, FullscreenOutline] }],
 })
 export class VisualizationFrameComponent {
   @Input() operatorId?: string;
+  @ViewChild("modalTitle") modalTitle!: TemplateRef<{}>;
   modalRef?: NzModalRef;
+  isFullscreen?: Boolean;
```

Comment:
make this a boolean instead of `boolean | undefined`. The value can either be true or false, and better to default to false.